File: Application/Models/Users/UserInformationDto.cs
namespace Application.Models.Users;

public record UserInformationDto(
    string Id,
    string? Email,
    string? Name,
    string? SecondName,
    string? LastName,
    string? IdentityNumber,
    string? CodeArea,
    string? PhoneNumber,
    string? ProfileImageUrl);

File: ClientApp/ClientApp/src/app/components/Account/Models/userInformationDto.ts
export interface UserInformationDto {
  id: string;
  email: string | null;
  name: string | null;
  secondName: string | null;
  lastName: string | null;
  identityNumber: string | null;
  codeArea: string | null;
  phoneNumber: string | null;
  profileImageUrl: string | null;
}

export interface UpdateUserInformationDto {
  name: string | null;
  secondName: string | null;
  lastName: string | null;
  identityNumber: string | null;
  codeArea: string | null;
  phoneNumber: string | null;
}
